app/models/project.rb

```diff
@@ -2,6 +2,20 @@
 class Project < ActiveRecord::Base
   using SymbolRefinements
 
+  #  BADGE_STATUS_CHOICE = %w(in_progress passing failing).freeze
+  class BadgeStatus
+    attr_reader :id, :name
+
+    def initialize(id, name)
+      @id = id
+      @name = name
+    end
+  end
```

Comment:
As remarked elsewhere, I need to keep this construct for the query condition on `projects.badge_status` to be generated by ransack. Using `attr_reader` and defining the initializer method allows for the array initialization in one step that comes next.